vumi_http_proxy/http_proxy.py

```diff
@@ -120,6 +121,8 @@ def read_file(self, blacklistfile):
         blacklist = []
         if not blacklistfile:
             log.err("No blacklist config file provided.")
+            log.err('Ending process.')
+            sys.exit()
```

Comment:
Exiting is probably a good idea, but perhaps not here. I think the solution is to move reading the config file out of `http_proxy` and into a helper function that we can call from `servicemaker` or `queen_of_ni`. Maybe remove this from this branch and we can make a separate issue for exiting if no blacklist is given?